camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/reporter/StatsdReporter.java

```diff
@@ -40,15 +45,28 @@ private void submitCountersToStatsd(Job job) throws IOException {
         }
     }
 
-    public static Boolean getStatsdEnabled(Job job) {
-        return job.getConfiguration().getBoolean(STATSD_ENABLED, false);
+    public static void logEventDelay(JobContext context, String topic, CamusWrapper wrap) {
+        if (getStatsdEnabled(context.getConfiguration())) {
+            long currentTime = System.currentTimeMillis();
+            long delay = currentTime - wrap.getTimestamp();
+            if ( delay >= 3600000 ) {
+                statsd = new NonBlockingStatsDClient(
+                    PREFIX, getStatsdHost(context.getConfiguration()), getStatsdPort(context.getConfiguration()), new String[]{"topic:" + topic}
+                );
+                statsd.gauge("load_delay", delay);
+            }
+        }
+    }
```

Comment:
I want to move to confluent'scamus where I'm building a proper sink to load data until then I think this can do. Proper way to initialize would require a complete rewrite of how reporting is done.